webpack.config.js

```diff
@@ -271,7 +274,8 @@ module.exports = async (env, argv) => {
       headers: {
         "Access-Control-Allow-Origin": "*"
       },
-      inline: !env.bundleAnalyzer,
+      hot: false,
+      inline: false,
```

Comment:
bling
![hotline](https://user-images.githubusercontent.com/4072106/98903723-0347bb80-246d-11eb-8895-c59455732d75.gif)
That can only mean one thing
![hotline2](https://user-images.githubusercontent.com/4072106/98903823-2d00e280-246d-11eb-97e7-dd2c202c1b2c.gif)